defrag/modules/docs.py

```diff
@@ -0,0 +1,108 @@
+import json
+from typing import Any, Dict, List
+from lunr import lunr
+from bs4 import BeautifulSoup
+from lunr.index import Index
+from defrag.modules.helpers.requests import Req
+from defrag.modules.helpers import Query, QueryResponse
+from defrag import app
+from pydantic import BaseModel
+
+
```

Comment:
Mod name is missing:
```suggestion

__MOD_NAME__ = "docs"

```